app/templates/components/dynamic-scan.emblem

```diff
@@ -72,10 +72,15 @@ if notVNCViewer
       .api-scanning
         div.warning-band.info-band
           .columns
-            .column.is-one-sixth
+            .column.is-one-sixth.align-middle
               = fa-icon 'info-circle'
             .column
               = t "modalCard.apiScan.dynamicScanText"
+        if file.minOsVersion
+          .alert.alert-info.margin-t-3q
+            | App requires minimum OS version
```

Comment:
@Praseetha-KR this needs to be added in locales so that we can get a Japanese translation of the message